x-pack/platform/packages/shared/kbn-elastic-assistant/impl/assistant/settings/assistant_settings_management.tsx

```diff
@@ -47,7 +47,19 @@ export const AssistantSettingsManagement: React.FC<Props> = React.memo(
     const {
       assistantFeatures: { assistantModelEvaluation: modelEvaluatorEnabled },
       http,
+      selectedSettingsTab: contextSettingsTab,
+      setSelectedSettingsTab,
     } = useAssistantContext();
+
+    useEffect(() => {
+      if (contextSettingsTab) {
+        // contextSettingsTab can be selected from Conversations > System Prompts > Add System Prompt
```

Comment:
I noticed this wasn't working in either `AssistantSettingsManagement` or `AIForSOCSettingsManagement` (which is based off of the former component), so I included a fix here.

Reply:
<img width="1355" alt="Screenshot 2025-04-08 at 12 00 16 PM" src="https://github.com/user-attachments/assets/ba547ab9-e8aa-4ce8-8b72-d6ea2f1c151a" />